### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#40] [General]
This is the start of the "Suggestions" channel. Here you can suggest features for the game or use [this form](https://forms.gle/8BSD1Udm6u8yABNA6) instead.

**Comment by SampleProvider** [comment]
Suggestion: Add this code.
```
document.querySelectorAll("button").forEach(function(item){
    item.addEventListener('focus',function(){
        this.blur();
    });
});
```

**Comment by SampleProvider** [comment]
spedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrunspedrun![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)
![image](https://user-images.githubusercontent.com/67865358/120901574-d8d86e00-c609-11eb-8309-8b302a4c6bc0.png)

**Reply by spsquared** [reply]
cool
